tests/CacheTest.php

```diff
@@ -35,41 +35,77 @@ public function it_can_cache_the_permissions()
     {
         $this->registrar->registerPermissions();
 
-        $this->assertCount(0, DB::getQueryLog());
+        if (config('permission.cache')) {
```

Comment:
The current tests should not be modified to test `config('permission.cache')`. Could you revert those changes.

I'm thinking that one test where you do some assertions on the amount of queries should be enough.